### PR TITLE
feat: add s3 and app version logs

### LINKED ISSUE #84: App Logs
As a developer, it would be useful to have some information logged for debugging purposes.

- [x] log the current version of the app on initial startup. 
- [x] log s3 url

### PR DESCRIPTION
**Pull request by Louise-X10**
closes #84

### CODE REVIEW

File: .env.template
Comment (mekhlakapoor) [inline .env.template:1]: add newline

**Review by mekhlakapoor — COMMENTED** [review]
newline error. also, also app version is "undefined"

File: .env.yaml.template [hunk 1]
```diff
@@ -1,3 +1,2 @@
 env_variables:
-  REACT_APP_S3_URL: 
-
+  REACT_APP_S3_URL:
```
Comment (jtyoung84) [inline .env.yaml.template:2]: It'd be cool if you can revert all changes done to this file in such a way that it doesn't show up in the PR

**Review by jtyoung84 — COMMENTED** [review]
Other than that one comment, it looks good to me. Please hold off merging it though so I can test that the aws amplify service picks up changes to the main branch.

**Comment by jtyoung84** [comment]
> Other than that one comment, it looks good to me. Please hold off merging it though so I can test that the aws amplify service picks up changes to the main branch.

Feel free to merge it whenever it's approved

**Pull request merged**